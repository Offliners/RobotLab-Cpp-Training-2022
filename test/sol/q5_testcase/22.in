2*18*17-6-6/8-1/16-(18*11/2*12)-12/16/4+16*18/7*1/11*20+8-11*4*18*15/20/7/20/12*11-13/11*16+7=
