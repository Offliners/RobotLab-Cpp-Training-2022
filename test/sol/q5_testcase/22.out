3
8
8
19
347
